src/util/drawing.js

```diff
@@ -0,0 +1,175 @@
+import external from '../externalModules.js';
+import { toolStyle, textStyle } from '../index.js';
+
+export function getNewContext (canvas) {
```

Comment:
JSDoc for all of these functions would be great. That helps people use the API because their editor will auto-suggest the arguments. It will also be auto-generated and displayed here: https://tools.cornerstonejs.org/api.html

(yeah I know it needs some reorganization...)

Reply:
Done.